jekyll/_sass/_sidebar.scss

```diff
@@ -15,8 +16,7 @@ $sidebar-spacing-base: 5px;
   height: 92vh;
   padding: 30px 24px 60px 15px;
   position: absolute;
-  bottom: 0;
```

Comment:
Absolute position isn't going to have much effect in this context without the `bottom` value. 

On tablet sized screens, whatever was contributing to the proper position on desktop goes away and I'm seeing a gap under the side nav (also the top of the sidenav is under the top nav): 
![Screen Shot 2021-04-26 at 11 28 23 AM](https://user-images.githubusercontent.com/5113432/116109164-8c048d80-a682-11eb-8b99-9014712f00fe.png)

Reply:
This needs to be removed to match the removal of the `top` rule below. The gap is due to the poorly-specified `92vh` height specification you called out elsewhere. All three of these rules (`top`, `bottom`, and `height`) never made sense to all be present, since there's no way for them to all be mutually applied.